payload/reggae/rules/cmake.d

```diff
@@ -86,7 +86,7 @@ private struct CompileOptions {
     SourceLanguage language;
 }
 
-private CompileOptions getCompileOptions(in imported!"std.json".JSONValue targetJsonObj, ulong compileGroupIndex) {
+private CompileOptions getCompileOptions(in imported!"std.json".JSONValue targetJsonObj, uint compileGroupIndex) {
```

Comment:
`size_t`? I don't know if that's the right answer, I'm asking.

Reply:
Yes, I think `size_t` is an even better option.